model/dc/registry.yaml

```diff
@@ -0,0 +1,65 @@
+groups:
+  - id: registry.dc
+    type: attribute_group
+    display_name: Datacenter Attributes
+    brief: >
+      Generic DataCenter infrastructure tagging.
+    attributes:
+      - id: dc.name
```

Comment:
it may be better to spell this out, check out the [attribute name abbreviation guidelines](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/attribute-naming.md#name-abbreviation-guidelines)